src/main/java/com/gmail/nossr50/commands/player/InspectCommand.java

```diff
@@ -74,7 +74,7 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
                 if (!isVanished && CommandUtils.tooFar(sender, target, Permissions.inspectFar(sender))) {
                     return true;
                 }
-
+                
```

Comment:
Why the extra dead space here? One return works to ensure it indents.

Reply:
Whoops, I had whitespace changes hidden. Fixed.

Reply:
Hehe ty